pkgs/tools/misc/lesspipe/default.nix

```diff
@@ -29,7 +29,7 @@ stdenv.mkDerivation rec {
 
   postInstall = ''
     for f in lesspipe.sh lesscomplete; do
-      wrapProgram "$out/bin/$f" --prefix-each PATH : "${lib.makeBinPath [ file gnused procps ]}"
+      wrapProgram "$out/bin/$f" --prefix-each PATH : "${lib.makeBinPath [ binutils file gnused procps ]}"
```

Comment:
This should be binutils-unwrapped because we don't need the wrapper and the extra dependenices.

Also please chime into #254409 which fixes the fundamental problem of missing deps.